deployment/default.js.j2

```diff
@@ -0,0 +1,57 @@
+const cloudflareConfig = require('./cloudflareConfig');
+const pDNSConfig = require('./PDNSConfig');
+const appsConfig = require('./appsConfig');
+
+module.exports = {
+  server: {
+    port: 16130,
+  },
+  explorer: 'https://explorer.runonflux.io',
+  fallbackexplorer: 'https://explorer.flux.zelcore.io',
+  database: {
+    url: '127.0.0.1',
+    port: 27017,
+    mainDomain: {
+      database: 'runonflux',
+      collections: {
+        // Collection of records associated with domain
+        records: 'records',
+      },
+    },
+  },
+  mainDomain: 'runonflux.io',
+  domainAppType: 'CNAME',
+  emailDomain: 'tessjonesie@gmail.com',
```

Comment:
shouldn't this be a runonflux.io email?

Reply:
Yup I guess it is better, will make the change. Any suggested email to use?

Reply:
It is an email that is used for obtaining certificate via certbot. It can basically be any mail even like placeholder I guess.
Later it receives expiry notification. 
We can use mine tadeas@runonflux.io and Ill setup some forwarding?